float32 left_setpoint
float32 right_setpoint
float32 cross_setpoint
float32 left_velocity
float32 right_velocity
float32 cross_velocity